content/influxdb/cloud/reference/release-notes/cloud-updates.md

```diff
@@ -14,6 +14,31 @@ aliases:
 InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
 To find information about the latest Flux updates in InfluxDB Cloud, see [Flux release notes](/influxdb/cloud/reference/release-notes/flux/).
 
+## October 2021
+
+### API invokable scripts
+
+Use [API invokable scripts](/influxdb/cloud/api-guide/api-invocable-scripts/) to assign and execute a script through an API endpoint.
+
+### Clockface v3
+
+Refresh look and feel of InfluxDB Cloud UI. Updated icons, fonts, and layouts to improve visibility, accessibility, and user experience.
+
+### Flux update
+
+Upgrade to [Flux v0.139](/flux/v0.x/release-notes/).
+
+### Cancellation flow
+
+Update the cancellation workflow.
```

Comment:
I'm leaning toward leaving this cancellation flow update out--is this important to users where they'll land now in case they cancel?

Reply:
I don't think we need to call this out in the changelog. Should we remove this point entirely?

Reply:
removing